plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleTableIndex.java

```diff
@@ -149,27 +140,28 @@ public void addColumn(OracleTableIndexColumn column)
 
     @NotNull
     @Override
-    public String getFullyQualifiedName(DBPEvaluationContext context)
-    {
-        return DBUtils.getFullQualifiedName(getDataSource(),
+    public String getFullyQualifiedName(DBPEvaluationContext context) {
+        return DBUtils.getFullQualifiedName(
+            getDataSource(),
             getTable().getContainer(),
-            this);
+            this
+        );
     }
 
     @Override
-    public String toString()
-    {
+    public String toString() {
         return getFullyQualifiedName(DBPEvaluationContext.UI);
     }
 
     @Override
     @Property(hidden = true, editable = true, updatable = true, order = -1)
     public String getObjectDefinitionText(DBRProgressMonitor monitor, Map<String, Object> options) throws DBException {
         if (indexDDL == null && isPersisted()) {
-            try (JDBCSession session = DBUtils.openMetaSession(monitor, this, "Read index definition")) {
-                indexDDL = JDBCUtils.queryString(session,"SELECT DBMS_METADATA.GET_DDL('INDEX', ?, ?) TXT FROM DUAL",
-                        getName(),
-                        getTable().getSchema().getName());
+            try (JDBCSession session = DBUtils.openUtilSession(monitor, this, "Read index definition")) {
```

Comment:
Why use util session? It's metadata read.
If you want to open a session if it doesn't exist, try this
```suggestion
            try (JDBCSession session = DBUtils.openUtilSession(monitor, this.getDataSource(), "Read index definition")) {
```